update nominator page (#271)
w3f/polkadot-wiki: docs/maintain-nominator.md

While your DOTs are staked by nominating a validator, they are 'locked' (bonded). You can receive new DOTs in your account but you cannot stake as validator or transfer DOTs away from your account. You can [un-nominate at any time](maintain-guides-how-to-unbond) to unlock your funds. Keep in mind that the un-nomination is effective in the next era.

> ### What does active/inactive nomination mean?

Reviewer: ```suggestion
### Active vs. Inactive Nomination
```

Reviewer: Headings should not be questions: https://writing.stackexchange.com/questions/14516/can-section-headings-in-a-paper-be-questions

When you go to the [Account actions](https://polkadot.js.org/apps/#/staking/actions) under staking page, you should see your bonded accounts and nomination

> status. If not, you can follow [this](maintain-guides-how-to-nominate-kusama) guide to configure it first. Your nominations will be effective in the next era, that means roughly 6 hours.

Reviewer 2: ```suggestion
status. If not, you can follow [this](maintain-guides-how-to-nominate-kusama) guide to configure it first. Your nominations will be effective in the next era, that means roughly 6 hours on Kusama and 24 hours on Polkadot.
```

![Nominations](/img/staking/polkadotjs-staking-account-actions.jpg)

> Suppose you have nominated five validator candidates, but 3 out of 5 were elected to 

Reviewer: ```suggestion
Suppose you have nominated five validator candidates, but three out of five were elected to 
```

Reviewer: It's best practice to use words for numbers below 10: https://www.grammarly.com/blog/when-to-spell-out-numbers/

> be the active validators then you should see 2 of your nominations are inactive. Active nomination means your nominated validators have been elected to be the active validator, whereas inactive means not get elected. 

Reviewer: ```suggestion
be the active validators, then you should see two of your nominations as inactive. Active nomination means your nominated validators have been elected to be in the active validator set, whereas inactive means they did not get elected. 
```

> ### What does inactive nominations could affect your rewards?

Reviewer: ```suggestion
### Possible effect of inactive nominations on rewards
```

Reviewer: Headings shouldn't be questions: https://writing.stackexchange.com/questions/14516/can-section-headings-in-a-paper-be-questions

> As long as you have nominated more than one validator candidates and have at least one of them got elected, your bonded stake will be fully distributed to the elected validators. That being said, you still will have the chance to get 0 rewards if you nominated very few validator candidates and no one got elected. So it would be safe to to choose as many trustworthy validators as you can (up to 16) to reduce the risk of getting nothing happened.

Reviewer 2: ```suggestion
As long as you have nominated more than one validator candidates and have at least one of them got elected, your bonded stake will be fully distributed to the elected validators. That being said, you still will have the chance to get 0 rewards if you nominated very few validator candidates and no one got elected. So it would be safe to to choose as many trustworthy validators as you can (up to 16) to reduce the risk of none of your nominated validators being elected.
```

> As long as you have nominated more than one validator candidates and have at least one of them got elected, your bonded stake will be fully distributed to the elected validators. That being said, you still will have the chance to get 0 rewards if you nominated very few validator candidates and no one got elected. So it would be safe to to choose as many trustworthy validators as you can (up to 16) to reduce the risk of getting nothing happened.

Reviewer: ```suggestion
As long as you have nominated more than one validator candidate and at least one of them got elected, your bonded stake will be fully distributed to the elected validators. That being said, you still will have the chance to get 0 rewards if you nominated very few validator candidates and no one got elected. So it would be safe to choose as many trustworthy validators as you can (up to 16) to reduce the risk of none of your nominated validators being elected.
```

> ### What you should take into consideration during nominating? 

Reviewer: ```suggestion
### What to take into consideration during nominating
```

![Staking Returns](/img/staking/polkadotjs-staking-returns.jpg)

> As a nominator, if you only want to know the profit of each validator has been made for each era, you can go to the [Returns](https://polkadot.js.org/apps/#/staking/returns) section under staking page by inputting the number of tokens you would like to stake to check it. And then nominate those who have a higher profit. However, that does not mean the right way to evaluate the validators' performance in overall.

Reviewer: ```suggestion
As a nominator, if you only want to know the profit each validator made for each era, you can go to the [Returns](https://polkadot.js.org/apps/#/staking/returns) section under the staking page by inputting the number of tokens you would like to stake to check it. Then, nominate those who have a higher profit. However, that does not guarantee the right way to evaluate the validators' overall performance.
```

> One example would be like if a validator regularly offline, their nominators most likely would get fewer rewards than the others. More importantly, when there are more than certain percentages of validators are [unreachable / unresponsiveness](learn-staking#unresponsiveness), those nominators who staked to the offline validators will be slashed (including validator itself).

Reviewer 2: The validator is always slashed before the nominators iirc. This is the reason why it is safer to nominate to highly self-staked validators over lower self-stake.

Reviewer 2: Nvm this was changed in https://github.com/paritytech/substrate/pull/3846

> One example would be like if a validator regularly offline, their nominators most likely would get fewer rewards than the others. More importantly, when there are more than certain percentages of validators are [unreachable / unresponsiveness](learn-staking#unresponsiveness), those nominators who staked to the offline validators will be slashed (including validator itself).

Reviewer: ```suggestion
One example would be if a validator is regularly offline, their nominators most likely would get fewer rewards than others. More importantly, when many validators are [unreachable](learn-staking#unresponsiveness), those nominators who staked with them will be slashed.
```

![Validator Stats](/img/staking/polkadotjs-staking-validator-stats.jpg)

> Thus, to be a smart nominator, it would be better to look at their [histories](https://polkadot.js.org/apps/#/staking/query/CmD9vaMYoiKe7HiFnfkftwvhKbxN9bhyjcDrfFRGbifJEG8) such as blocks produced, rewards & slashes and identity (if they have set) as well. Blocks produced means how many blocks they have been created for each era, while rewards & slashes indicate the total amount of tokens they have earned in each era. 

Reviewer: ```suggestion
Thus, to be a smart nominator, it would be better to look at their [histories](https://polkadot.js.org/apps/#/staking/query/CmD9vaMYoiKe7HiFnfkftwvhKbxN9bhyjcDrfFRGbifJEG8) such as blocks produced, rewards and slashes, and [identity](learn-identity) (if they have it set).
```

Reviewer: The part I removed is implied by the previous sentence.

> [Identity](learn-identity) is usually used to know who is behind running the validator. 

Reviewer: ```suggestion
```

> Moreover, you should do deep-dive research on the underlying validator candidates such as go over their websites to see who they are, what kind of infrastructure setup they are using, reputation, the vision behind the validator, and many more. 

Reviewer: ```suggestion
Moreover, a nominator should do comprehensive research on their validator candidates - they should go over the validators' websites to see who they are, what kind of infrastructure setup they are using, reputation, the vision behind the validator, and more. 
```

> To sum things up, instead of just by looking at the single indicator, combining with all these information together could form a more comprehensive metrics to evaluate the validators' capability.

Reviewer: ```suggestion
```

Reviewer: Superfluous
